app/src/main/res/drawable/elephant_error.xml

```diff
@@ -0,0 +1,106 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="200dp"
+    android:height="153dp"
+    android:viewportWidth="600"
+    android:viewportHeight="460">
+    <path
+        android:fillColor="#6D376D"
```

Comment:
you have to replace all these colors with `@color/...` because right now its not matching out color scheme. See `elephant_friend.xml`. This one is `@color/elephant_friend_border_color`

Reply:
Now I finally understand what you meant!
What do you propose to do with eyes and truck inners? Black?

Reply:
Also thing inside speech bubble, could be border or black
(sorry that I'm not independent enough in these things!)

Reply:
elephant_friend_border_color is supposed to be used as black